src/Parallel/CharmMain.tpp

```diff
@@ -111,6 +120,12 @@ void register_init_node_and_proc() noexcept {
   done_registration = true;
   // We explicitly register custom reducer functions first.
   _registerInitCall(register_custom_reducer_functions, 1);
+#ifdef DISABLE_OPENBLAS_MULTITHREADING
+  // Disable OpenBLAS multithreading since it conflicts with Charm++
+  // parallelism. Details:
+  // https://github.com/xianyi/OpenBLAS/wiki/Faq#multi-threaded
+  _registerInitCall(disable_openblas_multithreading, 1);
```

Comment:
Please move this to a `init_node_func`, as I requested initially.

Reply:
That's what I thought this is. Could you be more specific?

Reply:
In all the header files for the executables there's an `init_node` vector at the bottom

Reply:
Sure, are you saying you would prefer adding the `disable_openblas_multithreading` function to _every one_ of those vectors?

Reply:
Yes, because it is reasonable to want to disable it